mkdd_editor.py

```diff
@@ -2408,8 +2423,10 @@ def action_rotate_object(self, deltarotation):
         self.set_has_unsaved_changes(True)
         self.pik_control.update_info()
 
-    def action_ground_objects(self):
-        for pos in self.level_view.selected_positions:
+    def action_ground_objects(self, objects = None):
+        if objects is None:
+            self.level_view.selected_positions
```

Comment:
This statement has not effect here.

Guessing what your intention was, the pythonic solution would be:

```python
for pos in objects or self.level_view.selected_positions:
    ...
```